charts/galoy/values.yaml

```diff
@@ -830,6 +849,8 @@ kratos:
   secret:
     enabled: true
     nameOverride: ""
+  image:
+    tag: v0.13.0
```

Comment:
@krtk6160 

i had to add this kratos image tag back in. In my local dev charts environment the kratos container would not boot up properly and was trying to use by default an older version of kratos 0.11.0 that does not work with out current backend codebase. any ideas?

Reply:
@ntheile It's possible you have old charts in your `charts/charts/galoy/charts` directory. Delete all charts there, then run `helm dep up` in the `charts/charts/galoy` directory and retry.

Reply:
it worked thanks!